src/main/java/io/vertx/core/logging/impl/LogBackLogDelegateFactory.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2009 Red Hat, Inc.
+ * -------------------------------------
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ *
+ *     The Eclipse Public License is available at
+ *     http://www.eclipse.org/legal/epl-v10.html
+ *
+ *     The Apache License v2.0 is available at
+ *     http://www.opensource.org/licenses/apache2.0.php
+ *
+ * You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.core.logging.impl;
+
+/**
+ * 
+ * @author Patrick Sauts
+ *
+ */
+public class LogBackLogDelegateFactory implements LogDelegateFactory
+{
+   public LogDelegate createDelegate(final String clazz)
+   {
+      return new SLF4JLogDelegate(clazz);
```

Comment:
Well if someone is not aware that LogBack uses Slf4j. 
The point was mainly for users not to ask themselves too many questions.
And as it is a very small Class.
Let me know if you want me to remove it.